services/ux-backend/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	handler "github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	"k8s.io/klog/v2"
+)
+
+type serverConfig struct {
+	listenPort           int
+	tokenLifetimeInHours int
+}
+
+func loadAndValidateServerConfig() *serverConfig {
+	var config serverConfig
+
+	var err error
+	defaultTokenLifetimeInHours := 48
+	tokenLifetimeInHoursAsString := os.Getenv("ONBOARDING_TOKEN_LIFETIME")
+	if tokenLifetimeInHoursAsString == "" {
+		klog.Infof("No user-defined token lifetime provided, defaulting to %d ", defaultTokenLifetimeInHours)
+		config.tokenLifetimeInHours = defaultTokenLifetimeInHours
+	} else if config.tokenLifetimeInHours, err = strconv.Atoi(tokenLifetimeInHoursAsString); err != nil {
+		klog.Errorf("Malformed user-defined Token lifetime: %s. shutting down", tokenLifetimeInHoursAsString)
+		os.Exit(-1)
```

Comment:
These kinds of exists should be reserved for the top-most standalone invocation unit. As this is a standalone function it should covey issues by returning errors and the caller should make a decision about error logging, error handling, and program flow. I already mentioned all of these point in a previous PR regarding the generate token func.

To comply please have the following changes:
1. Instead of log, build and return an error using  `fmt.Errorf`
2. On the caller site, log the error and exist.